frontend/src/locales/scripts/utils.js

```diff
@@ -40,36 +40,40 @@ exports.setToValue = function setValue(obj, path, value) {
   o[a[0]] = value
 }
 
-function replacer(_, match) {
-  // Replace ###<text>### from `po` files with {<text>} in `vue`.
-  // Additionally, the old kebab-cased keys that can still be in the
-  // translations are replaced with camelCased keys the app expects.
-  // TODO: Remove `camel` and warning once all translation strings are updated.
-  // https://github.com/WordPress/openverse/issues/2438
-  if (match.includes("-")) {
-    console.warn("Found kebab-cased key in translation strings:", match)
-  }
-  return `{${kebabToCamel(match)}}`
-}
-
 /**
  * Replace ###<text>### with {<text>}.
  *
  * @param json {any} - the JSON object to replace placeholders in
+ * @param locale {string} - the locale of the JSON object
  * @return {any} the sanitised JSON object
  */
-const replacePlaceholders = (json) => {
+const replacePlaceholders = (json, locale) => {
   if (json === null) {
     return null
   }
+
+  function replacer(_, match) {
+    // Replace ###<text>### from `po` files with {<text>} in `vue`.
+    // Additionally, the old kebab-cased keys that can still be in the
+    // translations are replaced with camelCased keys the app expects.
+    // TODO: Remove `camel` and warning once all translation strings are updated.
+    // https://github.com/WordPress/openverse/issues/2438
```

Comment:
This can be converted into the function docstring.

Reply:
Thank you, done